xrpl4j-crypto-parent/xrpl4j-crypto-core/src/main/java/org/xrpl/xrpl4j/crypto/core/keys/Seed.java

```diff
@@ -116,7 +136,8 @@ public Seed(final UnsignedByteArray value) {
    */
   @VisibleForTesting
   Seed(final Seed seed) {
-    this.value = Objects.requireNonNull(seed.decodedSeed().bytes());
+    this.value = Objects.requireNonNull(seed)
+      .decodedSeed().bytes();
```

Comment:
I think this line is still untested. Can we add some unit tests for the copy constructor?

Reply:
Good catch - fixed!